docs/organizations/what-are-organizations.md

```diff
@@ -1,16 +1,18 @@
 # What are organizations
 
-Codacy organizations let you automatically import your Git provider organizations, repositories (including your personal repositories that don't belong to a Git provider organization), and team members into Codacy with a few clicks.
+Codacy organizations are an automatic import of your Git provider organizations, repositories (including your personal repositories that don't belong to a Git provider organization), and team members into Codacy with a few clicks.
```

Comment:
I agree more with what was before. It is not 'automatic' right away, it does let you automatically do it. But if we want to change, works for me